apps/meteor/ee/app/livechat-enterprise/server/api/lib/priorities.ts

```diff
@@ -37,11 +37,25 @@ export async function findPriority({
 	};
 }
 
-export async function updatePriority(_id: string, data: Pick<ILivechatPriority, 'name'>): Promise<ILivechatPriority> {
+export async function updatePriority(
+	_id: string,
+	data: Pick<ILivechatPriority & { reset: boolean }, 'name' | 'reset'>,
```

Comment:
`data: Pick<ILivechatPriority & { reset: boolean }, 'name' | 'reset'>`
change for
`data: Pick<ILivechatPriority, 'name'> & { reset: boolean }`